middleware.ts

```diff
@@ -1,61 +1,80 @@
 import { NextResponse } from 'next/server';
 import type { NextRequest } from 'next/server';
+import { verifyJWT, getJWTCookieName } from '@/lib/jwt-auth';
 
-const USERNAME = 'sws';
-const PASSWORD = 'sws';
+/**
+ * Middleware for JWT authentication
+ *
+ * DESIGN DECISION: Authentication (AuthN) vs Authorization (AuthZ)
+ *
+ * This middleware currently implements AUTHENTICATION ONLY:
+ * - Verifies user identity via JWT tokens from SAML SSO
+ * - Protects routes with the /protected/* prefix
+ * - Adds user identity headers for authenticated requests
+ *
+ * API routes (/api/*) are intentionally NOT protected because:
+ * 1. This branch implements AuthN only - AuthZ is future work
+ * 2. API routes currently return all data for a given subscription UUID
+ * 3. There's no user-data association or permission filtering yet
+ * 4. Main application routes (/, /applications/*) are also public
+ *
```

Comment:
Are you sure you want to do this right now?

By excluding /api from the middleware entirely, you are leaving endpoints like /api/acquia/applications completely open to the public internet on whatever environment this branch is on.

Reply:
This is addressed in AuthZ, https://github.com/SU-SWS/churro/tree/ACHOO-146. No intention of leaving API endpoints exposed.